### ISSUE TITLE
2023-09-12 OWL release defines PATO_0000070 as both a property and a class

### ISSUE THREAD

**Issue by jsstevenson** [monarch-initiative/mondo#6712]
I'm using the Python library `owlready2` to ingest the Mondo OWL release and run some SPARQL queries. In the most recent (2023-09-12) release, a change was introduced that causes the following error upon loading:

```
In [1]: import owlready2 as owl
In [8]: mondo = owl.get_ontology((Path(".") / "disease"/"data"/"mondo"/"mondo_2023-09-12.owl").absolute().as_uri()).load()
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[8], line 1
----> 1 mondo = owl.get_ontology((Path(".") / "disease"/"data"/"mondo"/"mondo_2023-09-12.owl").absolute().as_uri()).load()

File ~/code/disease-normalization/venv/lib/python3.11/site-packages/owlready2/namespace.py:1121, in Ontology.load(self, only_local, fileobj, reload, reload_if_newer, url, **args)
   1118 self._imported_ontologies._set(imported_ontologies)
   1120 # Search for property names -- must be done AFTER loading imported ontologies, because the properties might be partly defined in the imported ontologies
-> 1121 if self.world.graph.indexed: self._load_properties()
   1123 # Import Python module
   1124 global default_world, IRIS, get_ontology

File ~/code/disease-normalization/venv/lib/python3.11/site-packages/owlready2/namespace.py:1156, in Ontology._load_properties(self)
   1153 python_name_d = self.world._get_data_triple_sp_od(prop_storid, owlready_python_name)
   1155 if not isinstance(Prop, PropertyClass):
-> 1156   raise TypeError("'%s' belongs to more than one entity types (cannot be both a property and a class/an individual)!" % Prop.iri)
   1158 if python_name_d is None:
   1159   props.append(Prop.python_name)

TypeError: 'http://purl.obolibrary.org/obo/PATO_0000070' belongs to more than one entity types (cannot be both a property and a class/an individual)!
```

I am not an ontology person, but my very naive impression is that this is happening due to the addition of `PATO:0000070` (`amount`) as an `ObjectProperty` in the most recent OWL file:

```
1393:    <!-- http://purl.obolibrary.org/obo/PATO_0000070 -->
1394-                                                                                                                   
1395:    <owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/PATO_0000070"/>                                 
```

Apologies if this is intentional, and a fault of my OWL reader, not Mondo -- but figured I would put an issue here in case it's not (and if that's true, I'd be delighted if there are suggestions about better ways to grab this data in a Python environment). Thanks for your help!

**Comment by matentzn** [comment]
@twhetzel protocol for troubleshooting this kind of issue:

1. Run the imports pipeline locally, and check if the error persists by looking at `imports/merged_import.owl`.
1. ascertain the correct type of entity (should it be a class or a property?)
1. run a grep on the `mirror` directory to figure out which ontology is the culprit (`grep PATO_0000070 mirror/*.owl > issue.txt`
1. Once you isolate the ontology that has the wrong declaration, you proceed to making an issue on the respective ontology repository upstream (in this case, possibly maxo) to get it fixed
1. Now you can consider a temporary workaround. As this is a pretty important class, you cant just delete it, but you may have to hack the `mirror-maxo` (or whatever ontology goal) to remove the term until further notice. Add a comment above the goal which contains the name of the issue that will fix the problem

@jsstevenson thank you for taking the time to write the issue! Just because you seem to be interested in the issue, I want to point out that [having a class and an object property being the same IRI is allowed in OWL2](https://www.w3.org/TR/owl2-new-features/#F12%3a_Punning). So technically (again, not practically) speaking, the correct repo to file this error report would have been owlready2. I am glad though you filed it here, because the error was introduced by one of our dependencies and is clearly not intentional.

**Comment by jsstevenson** [comment]
> @jsstevenson thank you for taking the time to write the issue! Just because you seem to be interested in the issue, I want to point out that [having a class and an object property being the same IRI is allowed in OWL2](https://www.w3.org/TR/owl2-new-features/#F12%3a_Punning). So technically (again, not practically) speaking, the correct repo to file this error report would have been owlready2. I am glad though you filed it here, because the error was introduced by one of our dependencies and is clearly not intentional.

Thanks @matentzn ! I thought this might be the issue, but wanted to check just in case. Thanks for the help, all!

**Comment by matentzn** [comment]
Wow this brought me to a rabbit hole of issues in the end..

@twhetzel I have now dealt with all but 5, and I am inclined not to deal with 5 actually as I am 1 hour away from solving all of this upstream.

**Comment by twhetzel** [comment]
@matentzn ok, let me know if there is anything active for me to do other than grok your pending code changes.

**Comment by twhetzel** [comment]
@matentzn another similar issue with PATO was also reported today https://github.com/monarch-initiative/mondo/issues/6737

I gather fixing things upstream will take care of both of these reported issues, is that correct?

**Comment by matentzn** [comment]
Yes!

**Comment by matentzn** [comment]
If MAXO had a new release, you can go ahead and refresh the imports to see if the issue disappears.

**Comment by twhetzel** [comment]
Yes, it looks like there was a release in Oct. `<owl:versionInfo>2023-10-03</owl:versionInfo>`. @sabrinatoro can we review this (refresh imports) process and also make sure this fits with the existing work/changes being done now?

**Comment by twhetzel** [comment]
This is also now related to the PR #6864

**Comment by twhetzel** [comment]
Fixed with #6961
